### PR TITLE
Jerome sporks file

### PR DESCRIPTION
**Pull request by franklywatson**
Closes: #700 

## Description

* Restructure sporks.JSON for better query-ability and extract spork ID into numeric field
* Add missing Cadence tags 
______

For contributor use:

- [ X] Targeted PR against `master` branch
- [ X] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.

### CODE REVIEW

**Review by contil — APPROVED** [review]
LGTM

File: docs/content/node-operation/past-sporks.mdx [hunk 1]
```diff
@@ -249,7 +249,7 @@ While FCL v0.0.67 is not strictly necessary to use Mainnet 6, we strongly recomm
 | **Root Parent ID**: 5bc2b0900a5138e39d9209a8fe32e14b3e5c884bd36d2a645620f746b7c8bd47   |
 | **Root State Commit**: 6b9161a225b087a461ec95e710fdf4e73f6d6c9401ebf066207a021dced4ce5e |
 | **Git Commit**:                                                                        |
-| **Branch/Tag**: v0.8.2                                                                 |
```
Comment (Kay-Zee) [inline docs/content/node-operation/past-sporks.mdx:252]: why were these removed?

File: sporks.json [hunk 2]
```diff
@@ -1,950 +1,995 @@
...
+      "sporks": [
+        {
+          "id": 14,
+          "name": "Mainnet-14",
+          "spork-date": "2021-10-06T16:00:00Z",
+          "root-height": "19050753",
+          "root-parent-id": "ac4dbf344ce96e39e15081f1dc3fbbf6dc80532e402de9a57af847d3b35df596",
+          "root-state-commit": "641eb088e3ce1a01ff56df2d3a14372c65a7fef44c08799eb92cd7759d1d1d2a",
+          "git-commit-hash": "f019c1dbd778ce9f92dea61349ca36003678a9ad",
+          "state-artefacts" : {
```
Comment (Kay-Zee) [inline sporks.json:14]: interesting that we decided to go with this spelling of artefact

File: sporks.json [hunk 3]
```diff
@@ -1,950 +1,995 @@
...
-      "docker-tag": "v0.12.1",
-      "access-node": ""
+  "networks" : [
+    {
+      "name": "mainnet",
+      "sporks": [
+        {
+          "id": 14,
+          "name": "Mainnet-14",
+          "spork-date": "2021-10-06T16:00:00Z",
```
Comment (Kay-Zee) [inline sporks.json:9]: this includes the time, so maybe spork-time?

File: sporks.json [hunk 4]
```diff
@@ -1,950 +1,995 @@
...
+              "execution-state-wal-chunks": ""
+            }
+          },
+          "flow-go-tag": "v0.10.1",
+          "flow-dps-tag": "",
+          "cadence-tag": "v0.9.2",
+          "docker-tag": "v0.10.1",
+          "access-node": "access-001.mainnet1.nodes.onflow.org:9000"
+        },
+        {
```
Comment (Kay-Zee) [inline sporks.json:399]: haha, that's interesting interpretation, 0.9.

File: sporks.json [hunk 5]
```diff
@@ -1,950 +1,995 @@
...
+          "id": 0.9,
+          "name": "Candidate-9",
+          "spork-date": "2020-09-25T16:00:00Z",
+          "root-height": "6483246",
+          "root-parent-id": "9131733835702b0d6321088bddb4642a4964bb5c630440ccb0de47bdbc371d1a",
+          "root-state-commit": "90c6f406f5d21880d525ad4702cb249509b85e7f745db2de67e9fe541a56da4c",
+          "git-commit-hash": "",
```
Comment (Kay-Zee) [inline sporks.json:406]: any reason these are empty? just not that important since they're candidates?

File: sporks.json [hunk 6]
```diff
@@ -1,950 +1,995 @@
...
+              "execution-state-wal-chunks": ""
+            },
+            "s3": {
+              "root-checkpoint-file": "",
+              "root-protocol-state-snapshot": "",
+              "node-info": "",
+              "execution-state-wal-chunks": ""
+            }
+          },
+          "flow-go-tag": "",
```
Comment (Kay-Zee) [inline sporks.json:449]: and these as well

**Review by Kay-Zee — APPROVED** [review]

File: docs/content/node-operation/past-sporks.mdx (hunk 1 above)
Comment (franklywatson) [reply docs/content/node-operation/past-sporks.mdx:252]: I removed it because I couldn't find either a branch or a tag that has this version.

File: sporks.json (hunk 3 above)
Comment (franklywatson) [reply sporks.json:9]: Yeah this is the spork time, but adjusted for UTC given the time format

File: sporks.json (hunk 4 above)
Comment (franklywatson) [reply sporks.json:399]: Yeah, if we went with full numerics in the candidates then the other full Mainnet IDs would duplicate these and so numeric searches would have issues

File: sporks.json (hunk 5 above)
Comment (franklywatson) [reply sporks.json:406]: I couldn't find it in the Sporks history notion, and also there is no git commit hash for this version in the current public docs

File: sporks.json (hunk 6 above)
Comment (franklywatson) [reply sporks.json:449]: Same, again - there's nothing noted in the Sporks notion doc and nor in the current public docs for past sporks. Any suggestions you have to identify the missing data would be great

File: sporks.json (hunk 2 above)
Comment (franklywatson) [reply sporks.json:14]: Happy to change it LMK

File: sporks.json (hunk 3 above)
Comment (Kay-Zee) [reply sporks.json:9]: I meant the naming of the field

File: sporks.json (hunk 5 above)
Comment (Kay-Zee) [reply sporks.json:406]: for this particular field, since we have the git tag, we can get the commit hash if we just look at the commit that's tagged.

File: sporks.json (hunk 3 above)
Comment (franklywatson) [reply sporks.json:9]: Ah, fair point. Will do

File: sporks.json (hunk 5 above)
Comment (franklywatson) [reply sporks.json:406]: Ok thanks will do, my git-fu is a little rusty still
